xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/immutables/FluentCompareToTest.java

```diff
@@ -24,20 +25,28 @@ public void simpleTest() {
     assert (FluentCompareTo.is(SMALLER).lessThan(BIGGER));
     assert (FluentCompareTo.is(SMALLER).notGreaterThan(BIGGER));
     assert (FluentCompareTo.is(SMALLER).notGreaterThanEqualTo(BIGGER));
+    assertFalse(FluentCompareTo.is(SMALLER).greaterThan(SMALLER));
+    assertFalse(FluentCompareTo.is(SMALLER).lessThan(SMALLER));
 
     assert (FluentCompareTo.is(BIGGER).greaterThanEqualTo(BIGGER));
     assert (FluentCompareTo.is(BIGGER).greaterThanEqualTo(SMALLER));
     assert (FluentCompareTo.is(BIGGER).greaterThan(SMALLER));
     assert (FluentCompareTo.is(BIGGER).notLessThan(SMALLER));
     assert (FluentCompareTo.is(BIGGER).notLessThanOrEqualTo(SMALLER));
+    assertFalse(FluentCompareTo.is(BIGGER).greaterThan(BIGGER));
```

Comment:
These are fine for now, but idiomatically we prefer `assertThat` when possible over `assertFalse`.

Reply:
I thought, it would be better to not have the `!` inside the assert(), so I changed.

Reply:
You don't need `!` inside the assertion. It should look something like
```
assertThat(FluentCompareTo.is(BIGGER).greaterThan(BIGGER)).isFalse()
```